test/cov.jl

```diff
@@ -87,27 +100,32 @@ weight_funcs = (weights, aweights, fweights, pweights)
         @testset "Mean and covariance" begin
             (m, C) = mean_and_cov(X; corrected=false)
             @test m == mean(X, dims=1)
-            @test C == cov(X, dims=1, corrected=false)
+            @test C ≈ cov(X, dims=1, corrected=false)
```

Comment:
Any reason to use `≈` rather than `isequal`?

Reply:
Strict equality fails for sparse matrices because `cov` is specialized for `SparseMatrixCSC` in `Statistics` (https://github.com/JuliaLang/Statistics.jl/blob/b384104d35ff0e7cf311485607b177223ed72b9a/src/Statistics.jl#L1058), but `mean_and_cov` uses `covm` rather than `cov`.
I think if we want to achieve strict equality here, the fix should be in `Statistics` (specializing `covm` rather than `cov`)